code/modules/research/designs/design_weapons.dm

```diff
@@ -1,224 +1,234 @@
 	name = "chemsprayer"
 	desc = "An advanced chem spraying device."
 	id = "chemsprayer"
-	req_tech = list(TECH_MATERIAL = 3, TECH_ENGINEERING = 3, TECH_BIO = 2)
 	materials = list(MATERIAL_STEEL = 5000, MATERIAL_GLASS = 1000)
 	build_path = /obj/item/weapon/reagent_containers/spray/chemsprayer
-	sort_string = "TAAAA"
 
 /datum/design/item/weapon/rapidsyringe
 	name = "large capacity syringe gun"
 	id = "rapidsyringe"
-	req_tech = list(TECH_COMBAT = 3, TECH_MATERIAL = 3, TECH_ENGINEERING = 3, TECH_BIO = 2)
 	materials = list(MATERIAL_STEEL = 5000, MATERIAL_GLASS = 1000)
 	build_path = /obj/item/weapon/gun/launcher/syringe/rapid
-	sort_string = "TAAAB"
 
 /datum/design/item/weapon/large_grenade
 	name = "large chemical grenade case"
-	id = "large_Grenade"
-	req_tech = list(TECH_COMBAT = 3, TECH_MATERIAL = 2)
+	id = "large_grenade"
 	materials = list(MATERIAL_STEEL = 3000)
 	build_path = /obj/item/weapon/grenade/chem_grenade/large
-	sort_string = "TABAA"
+
 /* Re-add when the flashes are reworked.
 /datum/design/item/weapon/advancedflash
 	name = "advanced flash"
 	id = "advancedflash"
-	req_tech = list(TECH_COMBAT = 2, TECH_MAGNET = 2)
 	materials = list(MATERIAL_STEEL = 2000, MATERIAL_GLASS = 2000, MATERIAL_SILVER = 500)
 	build_path = /obj/item/device/flash/advanced
-	sort_string = "TADAA"
 */
+
 /datum/design/item/weapon/contactbeam
 	name = "C99 Supercollider Contact Beam"
 	id = "contactbeam"
-	req_tech = list(TECH_COMBAT = 5, TECH_MATERIAL = 5, TECH_POWER = 3)
 	materials = list(MATERIAL_STEEL = 2500, MATERIAL_GOLD = 1500, "uranium" = 1450)
 	build_path = /obj/item/weapon/gun/energy/contact
-	sort_string = "TAEAA"
 	price = 9000
 
 /datum/design/item/weapon/miningcutter
 	name = "210-V mining cutter"
 	id = "miningcutter"
-	req_tech = list(TECH_COMBAT = 1, TECH_MATERIAL = 3, TECH_POWER = 3)
 	materials = list(MATERIAL_STEEL = 7500, MATERIAL_GLASS = 2500)
 	build_path = /obj/item/weapon/gun/energy/cutter
-	sort_string = "TAEAB"
 	price = 2000
 
 /datum/design/item/weapon/plasmacutter
 	name = "211-V plasma cutter"
 	id = "plasmacutter"
-	req_tech = list(TECH_COMBAT = 3, TECH_MATERIAL = 4, TECH_POWER = 5)
 	materials = list(MATERIAL_STEEL = 15000, MATERIAL_GLASS = 5000, "gold" = 400)
 	build_path = /obj/item/weapon/gun/energy/cutter/plasma
-	sort_string = "TAEAC"
 	price = 4000
 
 /datum/design/item/weapon/advancedshield
 	name = "Advanced Combat Shield"
 	id = "advancedcombatshield"
-	req_tech = list(TECH_COMBAT = 5, TECH_MATERIAL = 6)
 	materials = list(MATERIAL_PLASTEEL = 10000, MATERIAL_GLASS = 2000)
 	build_path = /obj/item/weapon/shield/riot/advanced
-	sort_string = "TAEAD"
 
 /datum/design/item/weapon/divet
 	name = "Winchester NK Divet pistol"
 	id = "divet"
-	req_tech = list(TECH_COMBAT = 2, TECH_MATERIAL = 3)
 	materials = list(MATERIAL_STEEL = 10000, MATERIAL_PLASTEEL = 4000)
 	build_path = /obj/item/weapon/gun/projectile/divet
-	sort_string = "TACEA"
-
 	price = 4000
 
+/datum/design/item/weapon/divet/silenced
```

Comment:
this should not be fabricateable, its an antag specific item

Reply:
It can be researched only in illegal tech and in order to open them you need to research one of the illegal items